language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/DamlRecord.scala

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.javaapi.data
+
+import com.daml.ledger.api.v1.ValueOuterClass
+import org.checkerframework.checker.nullness.qual.NonNull
+import java.util._
+
+object DamlRecord {
```

Comment:
I am not exactly sure of what happened here, looks like for some reason I accidentally asked my IDE to port my code to Scala. Will delete.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/10132/commits/139f4d38ffcd64ea7da04062cc6c79fa418c8dac